bootstrap.py

```
@@ -0,0 +1,256 @@
+    print(f"[Bootstrap] {message}")
+
+
+def fail(message: str, code: int = 1) -> None:
+    print(f"[Bootstrap] ERROR: {message}", file=sys.stderr)
+    raise SystemExit(code)
+
+
+def resolve_command(command: List[str]) -> List[str]:
+    """Windows에서 .cmd/.bat 실행이 안정적으로 동작하도록 보정한다."""
+    if not command:
+        raise BootstrapError("빈 명령어는 실행할 수 없습니다.")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)